### PR TITLE
tpierce/508-onboarding-docs

### PR DESCRIPTION
**Pull request by 1Copenut**
Moving the request a 508 review template into this repo, moving in the 508 best practices, and updating the 508 issue template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/508-issue.md [hunk 1]
```diff
@@ -1,45 +1,52 @@
...
 ---
 
-<!--Issue Title 
-[ERROR TYPE]: 
-Brief description of error. I like to use the following for the error
-type. They are a help for searching and sorting issues.
+## Instructions
+
+1. Click or press the gear icon next to the "Labels" heading on the right. Search for your team in the labels list. If your team is not listed in the Labels menu, please leave a comment after you have created this issue. The Product Support team will create a new label for your team.
+2. If this issue is being opened by an external team, please add the `triage-incident` and `product support` labels.
```
Comment (rroueche) [inline .github/ISSUE_TEMPLATE/508-issue.md]: we don't use the term "external team" anymore. Did you mean VFS Team?

Comment (1Copenut) [reply .github/ISSUE_TEMPLATE/508-issue.md]: I did, will update immediately.

Comment (1Copenut) [reply .github/ISSUE_TEMPLATE/508-issue.md]: And updated

**Review by smoonpeyes — APPROVED** [review]
Looks good!

**Review by pjhill — APPROVED** [review]
Nice -- looks good

**Review by rroueche — APPROVED** [review]

**Pull request merged**